README.md

```diff
@@ -82,32 +82,52 @@ specific packages.
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/barrier.svg)](https://repology.org/project/barrier/versions)
 
-### FAQ
+## FAQ - Frequently Asked Questions
 
-Q: Does drag and drop work on linux?
+**Q: Does drag and drop work on linux?**
 
-A: No
+> A: No *(see [#855](https://github.com/debauchee/barrier/issues/544) if you'd like to change that)*
 
```

Comment:
Excellent, even better that you've linked to the issue. I think it might be worth adding a link to the Wayland and UTF-8 issues.

Reply:
I added a new answer at the end linking to #860 *(UTF-8)* and #109/#1251 *(Wayland)*, as neither seemed exclusively related to _'drag and drop'_ functionality.

Reply:
I do, yes, but your latest commit resolves my feedback. Cheers.